### PR TITLE
11 sep

### PR DESCRIPTION
**Pull request by Kapil0187**

### CODE REVIEW

File: app/views/flights/show_flight.html.erb
Comment (ShahirMansuri786) [inline app/views/flights/show_flight.html.erb:1]: please correct indent

File: app/views/home/show.html.erb
Comment (ShahirMansuri786) [inline app/views/home/show.html.erb:1]: please check indent here also

File: app/views/trains/show_train.html.erb
Comment (ShahirMansuri786) [inline app/views/trains/show_train.html.erb:1]: check indent in all files

Comment (Kapil0187) [reply app/views/trains/show_train.html.erb:1]: I corrected indentation in vs code but whenever I push code on github it changed.

File: app/controllers/bookings_controller.rb
[the remark this answers is not in the document]
Comment (Kapil0187) [reply app/controllers/bookings_controller.rb:1]: okay

**Pull request merged**